scout/load/hgnc_gene.py

```diff
@@ -103,9 +104,9 @@ def load_hgnc_genes(adapter, genes = None, ensembl_lines=None, hgnc_lines=None,
             ensembl_lines=ensembl_lines,
             hgnc_lines=hgnc_lines,
             exac_lines=exac_lines,
+            hpo_lines=hpo_lines,
             mim2gene_lines=mim2gene_lines,
             genemap_lines=genemap_lines,
```

Comment:
Trailing commas are scary. Could we just not have them? 😟

Reply:
I known theory they are allowed here, even being practical in some sense, but since they are not allowed in JSON and carry some unfortunate meaning with tuples, I get nervous. 😊

Reply:
Hehe, in python code I think they have the opposite effect. If you always include them it is more likely that nothing goes wrong.